src/EditorFeatures/CSharpTest/CrashHardTest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.UnitTests
+{
+    public class CrashHardTest
+    {
+        [Fact, Trait(Traits.Feature, "Crash")]
+        public void Crash()
+        {
+            ExceptionUtilities.FailFast(new Exception("Goodbye!"));
```

Comment:
Well, if this works, by the power of Git:

![you_didnt_see_anything](https://cloud.githubusercontent.com/assets/201340/13515033/e47fa962-e162-11e5-98cb-6c902f37ce05.gif)